### PR TITLE
8313710: jcmd: typo in the documentation of JFR.start and JFR.dump

### PR DESCRIPTION
**Pull request by kurashige23**
Hi,

I fixed the typos for JFR.start and JFR.dump.
Acconding to issue's description, there is some typo in JFR.stop documentation, but I couldn't find that. I confirmed that there is no such typo in this repository. So I thought there was no need to fix JFR.stop documentation.

I confirmed that the fixes are reflected and that all of the jdk_jfr tests pass.

Could someone please review it?

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8313710](https://bugs.openjdk.org/browse/JDK-8313710): jcmd: typo in the documentation of JFR.start and JFR.dump (**Bug** - P5)

### Reviewers
 * [David Holmes](https://openjdk.org/census#dholmes) (@dholmes-ora - **Reviewer**)
 * [Erik Gahlin](https://openjdk.org/census#egahlin) (@egahlin - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/16413/head:pull/16413` \
`$ git checkout pull/16413`

Update a local copy of the PR: \
`$ git checkout pull/16413` \
`$ git pull https://git.openjdk.org/jdk.git pull/16413/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 16413`

View PR using the GUI difftool: \
`$ git pr show -t 16413`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/16413.diff">https://git.openjdk.org/jdk/pull/16413.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/16413#issuecomment-1784741719)

### CODE REVIEW

**Comment by egahlin** [comment]
Looks like the added character and shift to the right will make the description exceed 80 characters in some lines, for example "included" for the begin option.

File: src/jdk.jcmd/share/man/jcmd.1
Comment (dholmes-ora) [inline src/jdk.jcmd/share/man/jcmd.1:1]: This file is generated from markdown sources that Oracle has to maintain. Please work with an Oracle engineer to get the changes applied to that source and then regenerate the `jcmd.1` nroff file. You can split this into a separate sub-task if needed.

**Review by dholmes-ora — CHANGES_REQUESTED** [review]

**Comment by kurashige23** [comment]
@egahlin 
> Looks like the added character and shift to the right will make the description exceed 80 characters in some lines, for example "included" for the begin option.

I'm sorry, I don't know from which letter to which letter is more than 80 characters, for example in line of begin option. Can you tell me that?
Also, I didn't know the rule that the number of characters per line should not exceed 80. If possible, could you please tell me where such a rule is stated?

File: src/jdk.jcmd/share/man/jcmd.1
Comment (kurashige23) [reply src/jdk.jcmd/share/man/jcmd.1:1]: Thank you for telling me about how to jcmd.1 nroff file is managed.
I will split this change into separate subtasks. Then if one of the Oracle engineers finds that subtask, will he/she work on it like any other bug?

Comment (dholmes-ora) [reply src/jdk.jcmd/share/man/jcmd.1:1]: Yes it will be assigned and addressed. Thanks

**Comment by egahlin** [comment]
> @egahlin
> 
> > Looks like the added character and shift to the right will make the description exceed 80 characters in some lines, for example "included" for the begin option.
> 
> I'm sorry, I don't know from which letter to which letter is more than 80 characters, for example in line of begin option. Can you tell me that? Also, I didn't know the rule that the number of characters per line should not exceed 80. If possible, could you please tell me where such a rule is stated?

You could look at line 319 in DCmdStart.java and 198 in DcmdDump.java. Make sure characters don't exceed. "...1234567890"

**Comment by kurashige23** [comment]
> > @egahlin
> > > Looks like the added character and shift to the right will make the description exceed 80 characters in some lines, for example "included" for the begin option.
> > 
> > 
> > I'm sorry, I don't know from which letter to which letter is more than 80 characters, for example in line of begin option. Can you tell me that? Also, I didn't know the rule that the number of characters per line should not exceed 80. If possible, could you please tell me where such a rule is stated?
> 
> You could look at line 319 in DCmdStart.java and 198 in DcmdDump.java. Make sure characters don't exceed. "...1234567890"

Thanks, I understand that characters should not exceed "... 1234567890" (88 characters) .  
I fixed line 204 in DcmdDump.java.  
In addition, the starting position of "...1234567890" in DcmdStart.java does not match the beginning of "Syntax :..", and it is hard to see visually whether characters exceed "...1234567890". So I fixed it.

File: src/jdk.jcmd/share/man/jcmd.1
Comment (kurashige23) [reply src/jdk.jcmd/share/man/jcmd.1:1]: Sorry for the late response.

I will ask my colleague to make it, so please wait a moment.

Comment (kurashige23) [reply src/jdk.jcmd/share/man/jcmd.1:1]: @dholmes-ora 
Sorry for the late response again.  
I issued sub-task.  
https://bugs.openjdk.org/browse/JDK-8324089

**Comment by kurashige23** [comment]
@dholmes-ora @egahlin 

I'm sorry that my slow response has prolonged this issue. I created a subtask and modified the source. If possible, please review them.

**Comment by dholmes-ora** [comment]
Thanks @kurashige23 , the manpage subtask will be handled by someone from Oracle. I'll leave it to JFR folk to do the review here.

**Comment by kurashige23** [comment]
Thanks @dholmes-ora , I understood that the subtask will be handled by someone from Oracle.

> I'll leave it to JFR folk to do the review here.

Does it mean that this pull request will not be merged until the subtask is resolved?

**Comment by kurashige23** [comment]
Could someone please review this PR?  
(Or will the review be done after https://bugs.openjdk.org/browse/JDK-8324089 is resolved?)

**Review by dholmes-ora — APPROVED** [review]
These changes seem fine to me.

Hopefully @egahlin can also approve.

**Comment by dholmes-ora** [comment]
BTW please merge your branch with master so that it is up to date. Thanks

**Comment by kurashige23** [comment]
I merged my branch with master. Thanks.

**Review by egahlin — APPROVED** [review]
Looks good. Thanks for fixing this. I can sponsor the change.

**Comment by kurashige23** [comment]
Thanks for review.

/integrate

**Pull request closed without merging**